test/jdk/javax/net/ssl/DTLS/PacketLossRetransmission.java

```diff
@@ -33,7 +33,7 @@
  * @build DTLSOverDatagram
  *
  * @run main/othervm PacketLossRetransmission client 0 hello_request
- * @run main/othervm PacketLossRetransmission client 1 client_hello
+ * @run main/othervm PacketLossRetransmission client 1 client_hello shortTimeOut
```

Comment:
Why are you only using a short time out for some messages?

(Also a small nit: line 109 has a typo.)

Reply:
Hi Matthew, only reducing the timeout when needed (most scenarios don't throw a socketTimeOut, only some), I'll add a PR description before removing the draft status